core/lib/types/src/storage/writes/compression.rs

```diff
@@ -0,0 +1,409 @@
+use super::*;
```

Comment:
do we need this? 
(I don't like the '*' inclusions, as it can be quite confusing on where the types are coming from)